testing/tests/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Installation steps to run API tests
+
+Run through the following steps to get API tests up and running on your Azure test bed.
+Checkout the following LME Branch on your Azure LME folder: cbaxley-122-testbed_from_scripts
```

Comment:
We will have to change this to "main" when we merge the168 branch.

Reply:
How long will it be before you think you'll be merging the 168 branch?  Is it worth changing now?

Reply:
Trying to merge it tomorrow. Should be merged before this ticket is merged though.

Reply:
Well, before the branch this pr is merging into.